rpm/sailfish-secrets.spec

```diff
@@ -94,6 +94,38 @@ Requires:   libsailfishcrypto = %{version}-%{release}
 %description -n libsailfishcrypto-tests
 %{summary}.
 
+
+%package -n libsailfishsecretscrypto
```

Comment:
No runtime requirement, but does BuildRequire glib etc.  Note that if the daemon isn't installed/running, calls will simply fail.  I could add Requires:   %{name} = %{version}-%{release} to ensure the daemon is pulled in, I suppose that would be more useful.  I'll do that.

Reply:
What do you mean by it does not require anything at runtime? What about Qt packages, for example (QtCore, QDBus, and so on)? Also, as you mentioned, the daemon package.. The rpm packager will add those anyway, actually. I think so. We may check this after having an rpm file. But it is unclear from reading what it needs for runtime. At the very least case, even if you don't link to anything and don't use any dependency, you link to the libstdc++ (if you don't produce a static-linked executable). Just my opinion - I'd add anything for the reader to understand what it needs at a glance.

Reply:
This particular package is a glib-based C library.  It doesn't require anything from Qt at runtime.  (It does require qmake at build time, I suppose.)  It does link against glib / gio.  Those are included as BuildRequires.  Am I misunderstanding?

Reply:
So if you dynamically linking to glib and gio, you should add both of them to the Require section, because rpm will check whether those are installed on the device.